### PR TITLE
cleanup formspecs + add translation support

### PR DESCRIPTION
**Pull request by Niklp09**
- ~~Add https://github.com/mt-mods/atm/issues/12~~
- Translation support
- ~~Cleanup the formspec code a bit~~

### CODE REVIEW

**Comment by Wuzzy2** [comment]
Use the opportunity to fix typos:

* Recepient → Recipient (multime strings)
* “Invalid amount \<@1\> : must be an integer number, aborting”: Remove the space after \>

**Review by BuckarooBanzay — COMMENTED** [review]
Looks good, but which PR will it be, this one or #14? :confounded: 

there_can_only_be_one

File: .luacheckrc [hunk 1]
```diff
@@ -1,5 +1,9 @@
 unused_args = false
 
+ignore = {
+	"631" -- Too long lines
```
Comment (wsor4035) [inline .luacheckrc]: this is bad, please properly fix

File: forms.lua [hunk 2]
```diff
@@ -1,173 +1,119 @@
...
-	"item_image_button[4.5,3;1,1;".. "currency:minegeld" ..";c-100;\n\n\b\b\b\b\b" .. "100" .."]" ..
-	"item_image_button[5.5,3;1,1;".. "currency:minegeld_5" ..";c-500;\n\n\b\b\b\b\b" .. "100" .."]" ..
-	"item_image_button[6.5,3;1,1;".. "currency:minegeld_10" ..";c-1000;\n\n\b\b\b\b\b" .. "100" .."]" ..
-	"list[current_player;main;0,4.25;8,1;]"..
-	"list[current_player;main;0,5.5;8,3;8]"..
-	"listring[]"..
-	default.get_hotbar_bg(0, 4.25)
-	minetest.after((0.1), function(gui)
-			return minetest.show_formspec(player:get_player_name(), "atm.form3", gui)
-		end, formspec)
```
Comment (wsor4035) [inline forms.lua]: perhaps this escape mess could be more neatly done? store in a variable, function wrap, etc `\n\n\b\b\b\b\b`

**Review by wsor4035 — COMMENTED** [review]
quick thoughts

**Review by BuckarooBanzay — APPROVED** [review]

**Pull request merged**